### PR TITLE
Add InexactError suggestion for Periodic

### LINKED ISSUE #77: better warning with `Yields.Periodic(0.12)`
offer suggsetion to user to say `Yields.Periodic(0.12,1)`

### PR DESCRIPTION
**Pull request by xlxs4**
Closes #77.

image

### CODE REVIEW

File: src/Yields.jl [hunk 1]
```diff
@@ -29,4 +29,15 @@ include("RateCombination.jl")
 include("NelsonSiegelSvensson.jl")
 
 include("precompiles.jl")
+
+
+function InexactError_hint(io::IO, ex::InexactError)
+    print(io, "\nSuggestion: try calling Periodic($(ex.val), 1) instead.")
```
Comment (alecloudenback) [inline src/Yields.jl:35]: Can we add the rationale? Something like:

> A `Periodic` rate requires also passing a compounding frequency. For examle, call `Periodic($(ex.val), 2)` for a rate compounded twice per period.

**Review by alecloudenback — CHANGES_REQUESTED** [review]
Thanks, can you make the one change in the comment?

File: src/Yields.jl (hunk 1 above)
Comment (xlxs4) [reply src/Yields.jl]: Sure thing!

**Comment by alecloudenback** [comment]
Thanks!

**Comment by xlxs4** [comment]
Great ^^ Since you've tagged the issue with https://github.com/JuliaActuary/Yields.jl/labels/Hacktoberfest, know that for the repository/PR to be participating, you either have to add the hacktoberfest topic in the repository, or tag the PR with the hacktoberfest-accepted label

**Comment by alecloudenback** [comment]
@xlxs4 did I add the tag correctly?

**Comment by xlxs4** [comment]
Yes, thanks a lot :)

**Pull request merged**